### PR TITLE
Fixes #151, BernouilliNB converter

### LINKED ISSUE #151: onnxruntime for converted BernouilliNB, MultinomialNB (scikit-learn) does not produce the same results as the original model

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: onnxmltools/convert/sklearn/operator_converters/NaiveBayes.py [hunk 1]
```diff
@@ -5,8 +5,10 @@
 # --------------------------------------------------------------------------
 
 from ....proto import onnx_proto
-from ...common._apply_operation import apply_exp, apply_reshape, apply_sub
+from ...common._apply_operation import apply_exp, apply_reshape, apply_sub, apply_max, apply_min
```
Comment (wenbingl) [inline onnxmltools/convert/sklearn/operator_converters/NaiveBayes.py]: Did max and min op need broadcasting support?

Comment (xadupre) [reply onnxmltools/convert/sklearn/operator_converters/NaiveBayes.py]: It would have been better.

**Comment by wenbingl** [comment]
@xadupre, is this PR still needed?

**Comment by xadupre** [comment]
Yes we can drop it.

**Pull request closed without merging**